docs/reference/sections/dom-binding.md

```diff
@@ -1,6 +1,21 @@
-Full documentation for the DOM Binding module is coming soon.
+DOM Binding is powered by Rivets.js, providing a flexible templating approach to cart rendering.
 
-In the meantime, you can check out the example on the [front page][] in combination with the [Rivets.js documentation][] to get an idea of how things work.
+See the [DOM Binding] section in the Guide for more.
 
-[front page]: /
-[Rivets.js documentation]: http://rivetsjs.com/
+### Rivets Binders
+
+A full breakdown of Rivets' available binders can be found under the Rivets docs [Binder Reference][].
```

Comment:
Good point. We don't want to create too much work for ourselves. This could even just be a single sentence along the lines of "there are a number of binders you'll be working with, including...", and then link out to the Rivets docs.